code/modules/admin/verbs/adminpm.dm

```diff
@@ -531,7 +531,7 @@
 	if(!log_message)
 		return
 
-	var/raw_message = log_message
+	var/raw_message = sanitize(trim(log_message))
```

Comment:
This feels wrong, I feel like you should just change the usage of it instead, otherwise it's no longer the "raw message" that it's meant to be.

Reply:
Fixed